src/GeoJSON.jl

```diff
@@ -48,11 +52,11 @@ function geojson end
 for geom in (:AbstractFeatureCollection, :AbstractGeometryCollection, :AbstractFeature,
         :AbstractMultiPolygon, :AbstractPolygon, :AbstractMultiLineString,
         :AbstractLineString, :AbstractMultiPoint, :AbstractPoint)
-    @eval geojson(obj::GeoInterface.$geom) = JSON.json(geo2dict(obj))
+    @eval geojson(obj::GeoInterface.$geom) = JSON3.write(geo2dict(obj))
```

Comment:
Can we rename this as `write(obj::GeoInterface.$geom)`, deprecate `GeoJSON.geojson(input)`, and point people to `GeoJSON.write(input)` instead (for consistency with `JSON3`).